kubernetes-model-generator/openshift-model/src/test/java/io/fabric8/openshift/api/model/DeploymentConfigRollbackTest.java

```diff
@@ -0,0 +1,74 @@
+package io.fabric8.openshift.api.model;
+
+import static net.javacrumbs.jsonunit.core.Option.IGNORING_ARRAY_ORDER;
+import static net.javacrumbs.jsonunit.core.Option.IGNORING_EXTRA_FIELDS;
+import static net.javacrumbs.jsonunit.core.Option.TREATING_NULL_AS_ABSENT;
+import static net.javacrumbs.jsonunit.fluent.JsonFluentAssert.assertThatJson;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.fabric8.kubernetes.api.model.Helper;
+import io.fabric8.kubernetes.api.model.ObjectReference;
+import java.util.Collections;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class DeploymentConfigRollbackTest {
+
+  private final ObjectMapper mapper = new ObjectMapper();
+
+  @Test
+  void deploymentConfigRollbackTest() throws Exception {
+    // given
+    final String originalJson = Helper.loadJson("/valid-deploymentConfigRollback.json");
+
+    // when
+    final DeploymentConfigRollback deploymentConfigRollback = mapper.readValue(originalJson,
+      DeploymentConfigRollback.class);
+    final String serializedJson = mapper.writeValueAsString(deploymentConfigRollback);
+
+    // then
+    assertThatJson(serializedJson).when(IGNORING_ARRAY_ORDER, TREATING_NULL_AS_ABSENT, IGNORING_EXTRA_FIELDS)
+      .isEqualTo(originalJson);
+  }
+
+  @Test
+  void deploymentConfigRollbackBuilderTest() throws Exception {
+    //given
+    final DeploymentConfigRollback expected = mapper.readValue(
+      Helper.loadJson("/valid-deploymentConfigRollback.json"),
+      DeploymentConfigRollback.class);
+
+    //when
+    DeploymentConfigRollback actual = new DeploymentConfigRollbackBuilder()
+      .withName("app-to-rollback")
+      .withSpec(
+        new DeploymentConfigRollbackSpecBuilder().withFrom(
+            new ObjectReference(
```

Comment:
Why not use ObjectReferenceBuilder here?

Reply:
Oversight, will update.